tests/retryable-reads/retryable-reads-001.phpt

```diff
@@ -0,0 +1,58 @@
+--TEST--
+Retryable reads: supported single-statement operations include transaction IDs
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php skip_if_not_libmongoc_crypto(); ?>
+<?php skip_if_server_version('<', '3.6'); ?>
+<?php skip_if_no_failcommand_failpoint(); ?>
+<?php skip_if_not_clean(); ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+class Observer implements MongoDB\Driver\Monitoring\CommandSubscriber
+{
+    public function commandStarted(MongoDB\Driver\Monitoring\CommandStartedEvent $event)
+    {
+        printf("Command started: %s\n", $event->getCommandName());
+    }
+
+    public function commandSucceeded(MongoDB\Driver\Monitoring\CommandSucceededEvent $event)
+    {
+    }
+
+    public function commandFailed(MongoDB\Driver\Monitoring\CommandFailedEvent $event)
+    {
+    }
+}
+
+$manager = new MongoDB\Driver\Manager(URI, ['retryReads' => true]);
+
+$bulk = new MongoDB\Driver\BulkWrite;
+$bulk->insert(['x' => 1]);
+$bulk->insert(['x' => 2]);
+
+$manager->executeBulkWrite(NS, $bulk);
+
+configureFailPoint($manager, 'failCommand', ['times' => 1], ['failCommands' => ['find'], 'closeConnection' => true]);
+
+$observer = new Observer;
+MongoDB\Driver\Monitoring\addSubscriber($observer);
+
+$command = new MongoDB\Driver\Command([
+    'find' => COLLECTION_NAME,
+    'filter' => ['x' => 1],
+]);
+$cursor = $manager->executeReadCommand(DATABASE_NAME, $command);
```

Comment:
Any reason not to use `executeQuery()` in this test? Perhaps we can split this test into two and have the other utilize `MongoDB\Driver\Query` with `executeQuery()`? The `executeReadCommand()` test can continue to use the `find` command, but alternatively we could run an aggregate for count (effectively `countDocuments` CRUD method).

Reply:
The test is now split, there is one each for `executeReadCommand` and `executeQuery`.